src/stock-sources/stock-sources-delete/stock-sources-delete.test.tsx

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import { render, fireEvent, waitFor, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { showModal, showSnackbar } from "@openmrs/esm-framework";
+import { deleteStockSource } from "../stock-sources.resource";
+import StockSourcesDeleteActionMenu from "./stock-sources-delete.component";
```

Comment:
we can remove unused imports

Reply:
@jabahum This import is being used, which one in specific did you mean